src/main/resources/assets/ping/lang/en_us.json

```diff
@@ -3,15 +3,15 @@
   "ping:key.categories.ping": "Ping",
 
   "_comment": "Pings",
-  "key.ping": "Opens Ping menu",
-  "ping.key.alert": "Ping (Alert)",
+  "key.ping": "Open pinging menu",
+  "ping.key.alert": "Ping (Attention!)",
   "ping.key.mine": "Ping (Mine)",
   "ping.key.look": "Ping (Look)",
-  "ping.key.goto": "Ping (Go to)",
+  "ping.key.goto": "Ping (Go here)",
 
   "_comment": "Subtitles",
-  "ping.subtitle.bloop": "Ping bloops",
+  "ping.subtitle.bloop": "Ping bloop",
 
   "_comment": "Gui",
   "ping.pingSelect.title" : "Ping Select"
```

Comment:
Is this string used somewhere? I think it was supposed to be placed above these 4 types of pings. If so, I would propose to change the existing text to e.g. "Select ping type". If I'm wrong, please let me know.